setup.py

```diff
@@ -12,12 +12,31 @@
 needs_pytest = {'pytest', 'test', 'ptr'}.intersection(sys.argv)
 pytest_runner = ['pytest-runner'] if needs_pytest else []
 
+# Check Python version
+if sys.version_info[:2] < (3, 6):
+    raise RuntimeError('Python version >=3.6 required.')
+
+# Add release
+RELEASE = "0.0.0-dev0"
+
+# Check numpy is installed
 try:
+    import numpy as np
+except ImportError:
+    print("MembraneCurvature requires NumPy.")
```

Comment:
@tylerjereddy @IAlibay @orbeckst I forget why MDAnalysis has all this numpy complexity -- something to do with setuptools not finding the right `include` directory for compilation? Do you think @ojeda-e can just skip this check and leave it in `install_requires` if she's not compiling any extensions?

Regardless, I would leave this as an `ImportError`, possibly with a more informative error message. At this point `setup.py` just prints the message and keeps going. The original MDAnalysis setup.py has `sys.exit(-1)` [afterwards which quits Python with an error code](https://github.com/MDAnalysis/mdanalysis/blob/8f9f402fe65d6a90e88b60b6da735c130425f00e/package/setup.py#L195), but IMO this is what the error is for.

Reply:
I don't _think_ downstream packages need to ensure numpy import. Putting it in `install_requires` should be fine (assuming the minimum/maximum version differs from upstream MDAnalysis).

Reply:
I will get rid of this, then. Thanks.